client/app/optionsets/landuse-form.js

```diff
@@ -96,6 +96,28 @@ export const DCPINDICATETYPEOFFACILITY = {
   },
 };
 
+export const DCPRELATEDACQUISITIONOFPROPERTYOPT = {
```

Comment:
hmm this looks just like `COMMON_OPTIONSETS.YES_NO_INTEGER`

Reply:
true!